restarter/main.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"os/exec"
+)
+
+func main() {
+	http.HandleFunc("/restart", restartSignature)
+
+	var host, port string
+	flag.StringVar(&host, "host", "0.0.0.0", "Listen host")
+	flag.StringVar(&port, "port", "8081", "Listen port")
+	flag.Parse()
+
+	address := host + ":" + port
+
+	log.Fatal(http.ListenAndServe(address, nil))
```

Comment:
This is kinda ugly. Extract the important pieces from the log.

Reply:
fixed